### ISSUE TITLE
[Issue] Ranks reset if server goes down

### ISSUE THREAD

**Issue by UWPA** [rogeraabbccdd/Kento-Rankme#117]
Can this be sorted? If the server drops or lags, then it just resets the person's rank rather than inserting to the last amount. This has happened at least 2-3 times a day and is becoming a pain in the ass because people just cry about it.

**Comment by TandelK** [comment]
That might be due to Database and Server disconnection issue. Check your connection between Database and game Server

**Comment by UWPA** [comment]
> 
> 
> That might be due to Database and Server disconnection issue. Check your connection between Database and game Server

Yes, that's what I mean. I have 19 servers with, most of the time, all being 10/10 with the occasional 9 or 8/10, but due to this plugin having constant open connections, if the game/web server lags for a minute, it nests up and can occasionally choke it. Restarting then causes the loss of rank.
